nx.json

```diff
@@ -16,21 +16,43 @@
 			"inputs": ["production", "^production"]
 		},
 		"lint": {
-			"inputs": ["default", "{workspaceRoot}/.eslintrc.json"]
+			"inputs": ["default"]
 		},
 		"test": {
 			"inputs": ["default", "^production", "{workspaceRoot}/jest.preset.js"]
+		},
+		"@nx/esbuild:esbuild": {
+			"cache": true,
```

Comment:
not sure if we should cache builds on production, i think it might be better to build fresh?